pkg/controller/storagecluster/reconcile.go

```diff
@@ -33,6 +34,9 @@ type ReconcileStrategy string
 // StorageClassProvisionerType is a string representing StorageClass Provisioner. E.g: aws-ebs
 type StorageClassProvisionerType string
 
+// DeviceTypes is list of valid device type from which storagecluster can be created
+var DeviceTypes = []string{"nvme", "ssd", "hdd"}
```

Comment:
This is still not done, make them constants as per https://github.com/openshift/ocs-operator/pull/859

Reply:
I went with this approach https://github.com/openshift/ocs-operator/pull/859#discussion_r516121654.  Not against of adding const it is just that there will be devices which will be added in future as of now PCI will be one. creating const for each one. and array of devices which will better? and there will one benefit of using const that will be when we will tune the osd. So i am like 5050 on both approaches .

Reply:
I am 100% for consts, so since you're not fully against it I'd say go ahead with the change. :)

Reply:
consts it is. Added.